### PR TITLE
Add helper function to get field index from vector layer

### PR DESCRIPTION
**Pull request by Joonalai**
Uses standard KeyError if the field is not found.

### CODE REVIEW

File: tools/layers.py [hunk 1]
```diff
@@ -143,3 +146,22 @@ def evaluate_expressions(
...
+def get_field_index(layer: QgsVectorLayer, field_name: str) -> int:
+    """
+    Get field index if exists
+    :param layer: QgsVectorLayer
+    :param field_name: name of the field
+    :return: index of the field
+    """
+    field_index = layer.fields().indexFromName(field_name)
+    if field_index == -1:
+        raise QgsPluginLayerException(
```
Comment (LKajan) [inline tools/layers.py:160]: Could we raise just a standard KeyError instead of a custom one?

**Review by LKajan — APPROVED** [review]

**Review by LKajan — CHANGES_REQUESTED** [review]
could you`still check and fix those black and isort issues

File: tools/layers.py (hunk 1 above)
Comment (Joonalai) [reply tools/layers.py:160]: Good idea!

**Review by Rikuoja — APPROVED** [review]
LGTM!

**Pull request merged**